middleware/csp.ts

```diff
@@ -5,7 +5,7 @@ import { isClient } from '@vueuse/core';
  * and force the route to reload from the server.
  */
 export default defineNuxtRouteMiddleware((to, from) => {
-  if (isClient && from.path === '/checkout/payment-method') {
+  if (isClient && from.path === '/checkout/pay/method') {
     abortNavigation();
     window.location.href = to.fullPath;
```

Comment:
can we do a change here, I noticed that this actually appears a bit weird.
```suggestion
  if (isClient && from.path === '/checkout/pay/method') {
     setTimeout(() => {
      window.location.href = to.fullPath;
    }, 100);
    return abortNavigation();
```

The component loads and then it does hard refresh and reloads and it does not look good.

This change at least for me appears to fix it can you also verify?

Reply:
okay, will check as well.